com/truckmuncher/api/trucks.proto

```diff
@@ -30,11 +30,20 @@ service TruckService {
     /**
      * Modify the serving mode for a truck.
      *
-     * This call requires Vendor authorization
+     * This call is an alias for reportServingMode
+     * This call is deprecated and should no longer be used by clients. It will be removed in the near future.
+     * This call requires the user to be logged in
      */
     rpc modifyServingMode (ServingModeRequest) returns (ServingModeResponse);
```

Comment:
Why is this deprecated? Are trucks just reporting their own location with the other route and then the API knows that they own the truck, so automatically verify it?